google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -589,6 +589,15 @@ public Builder setSystemExecutorProvider(ExecutorProvider executorProvider) {
       return this;
     }
 
+    /**
+     * Sets the number of pullers used to pull messages from the subscription. Defaults to the
+     * number of available processors.
+     */
+    public Builder setNumPullers(int numPullers) {
```

Comment:
This naming is a little opaque (for Java). Maybe: `setParallelPullCount` or something like that.